server/controllers/cohort.js

```diff
@@ -218,11 +219,21 @@ const deleteCohortById = (req, resp) => {
       if (lists) {
         const listsIds = lists.map(lists => lists._id);
 
-        return Comment.deleteMany({ listId: { $in: listsIds } });
+        return Comment.updateMany(
+          { listId: { $in: listsIds } },
```

Comment:
Shouldn't the listIds be sanitized as well?

Reply:
Those listsIds are coming from the database, not from the request of the client, so in my opinion, it doesn't have to be sanitized. Those ids are generated by MongoDB itself, so there will be no query injections.

Reply:
Ok